### DISCUSSION TITLE
2023-10-13

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#646] [daily_meeting]
### 내일의 할 일

**Comment by aak2075** [comment]
### 체크인

10점 - 벌써 불금이네요

**Comment by jaehee329** [comment]
### 체크인
8점 - 오늘만 지나면 아침잠좀 잘 수 있겠네요

**Comment by nlom0218** [comment]
### 체크인

5점 - 알고리즘이 안 풀려요 왜죠?

**Comment by woo-jk** [comment]
### 체크인
9점 - 내일은 늦잠 좀 자야겠어요

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 인적성 검사 불합 확률은 매우 낮다고 하네요... 면접 준비도 열심히 해봐야겠습니다.

**Comment by woosung1223** [comment]
### 체크인
7점 - 오늘은 금요일이지만 피곤해서 -3점입니다
